xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/dropdown-menu.html

```diff
@@ -1,26 +1,96 @@
 <template>
-  <div class="livedata-dropdown btn-group">
+  <div class="livedata-topbar">
+
+    <!-- Drowpdown -->
+    <div class="livedata-dropdown btn-group">
+
+      <!-- Drowpdown button-->
+      <a class="btn btn-default dropdown-toggle" title="More Actions" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true" role="button">
+        <span class="fa fa-ellipsis-v"></span>
+      </a>
+
+      <!-- Drowpdown content-->
+      <ul class="dropdown-menu">
+
+        <!-- Change layout-->
+        <li class="dropdown-header">Change Layout</li>
+        <li
+          v-for="layout in data.meta.layouts"
+        >
+          <a href="#" @click.prevent="logic.changeLayout(layout)">
+            <span :class="data.meta.layoutDescriptors[layout].icon"></span>
+            {{ data.meta.layoutDescriptors[layout].name }}
+          </a>
+        </li>
+
+        <!-- Common actions -->
+        <li class="dropdown-header">Actions</li>
+        <li>
+          <a href="#" @click.prevent="sortPanelOpen = !sortPanelOpen">
+            <span class="fa fa-sort"></span> Toggle Sort Panel
```

Comment:
That was what we agreed for the jira issue, to also implement a sort option in the hamburger menu. I reckon that for now it is not very helpful as it's single sort, but it still add a default sort system for the card layout. For now we could disable it in the table layout if you want though.

Reply:
What I don't like is the sort panel that takes space from the layout. Having a sort group in the drop down menu that lists the properties and shows the selected one with a check icon is fine with me.

Reply:
That's what we agreed at the end of the call with Vincent. If you are still not satisfied with this we can try to rework the specs, but this might take more time.

Reply:
I really don't remember agreeing on a **sort panel** that you show / hide from the drop down menu.